lib/fs.js

```diff
@@ -2379,8 +2379,13 @@ WriteStream.prototype.open = function() {
 
 
 WriteStream.prototype._write = function(data, encoding, cb) {
-  if (!(data instanceof Buffer))
-    return this.emit('error', new Error('Invalid data'));
+  if (!(data instanceof Buffer)) {
```

Comment:
Should likely update this to use `Buffer.isType()` but let's check with @mcollina first

Reply:
Ok let me know and I'll change it.

Also, where can I see the CI build?
I think my tests are passing but I want to see the official CI.

Reply:
I would keep this as it is, with `instanceof`. `Buffer.isBuffer` is mainly needed outside of core.